Program.cs

```diff
@@ -1,15 +1,16 @@
-﻿using System;
+﻿using FoodDelivery21.Data;
+using FoodDelivery21.Service;
+using FoodDelivery21.UI;
+using System;
 
-namespace FoodDelivery
+namespace FoodDelivery21
```

Comment:
21 is a year of creation this app. It makes naming of service more unique because there are many similar apps but not all of them were created in this year.